run-tests.php

```diff
@@ -1091,7 +1091,7 @@ function find_files(string $dir, bool $is_ext_dir = false, bool $ignore = false)
         }
 
         // Otherwise we're only interested in *.phpt files.
-        if (substr($name, -5) == '.phpt') {
+        if (substr($name, -5) == '.phpt' && substr($name, 0, 2) !== '.#' ) {
```

Comment:
maybe introduce a constant somewhere, so this magic constant gets somehow readable?